### ISSUE TITLE
Kucoin - Errors caused by BCHSV symbol not updated as BSV

### ISSUE THREAD

**Issue by glenntomas** [CoinAlpha/hummingbot#3648]
**Describe the bug**
When creating Avellaneda MM bot and use `BCHSV-USDT` as market it throws an error `List index out of range`

**Steps To Reproduce**
1. Create Avellaneda MM strategy
2. Enter `BCHSV-USDT` as your market

**Screenshots**
- `version 0.40.0` throws an error `List index out of range`
- Both Pure Market Making and Avellaneda MM throws the same error

![image](https://user-images.githubusercontent.com/73885708/123731829-89fdbb80-d8cb-11eb-9982-7a9e94f8b49d.png)

![image](https://user-images.githubusercontent.com/73885708/123737914-67bd6b00-d8d6-11eb-9a7b-cce61b244df8.png)

```
Traceback (most recent call last):
  File "/home/hummingbot/hummingbot/core/utils/async_utils.py", line 9, in safe_wrapper
    return await c
  File "/home/hummingbot/hummingbot/client/command/create_command.py", line 63, in prompt_for_configuration
    await self.prompt_a_config(config)
  File "/home/hummingbot/hummingbot/client/command/create_command.py", line 106, in prompt_a_config
    prompt = await config.get_prompt()
  File "/home/hummingbot/hummingbot/client/config/config_var.py", line 54, in get_prompt
    return await self.prompt()
  File "/home/hummingbot/hummingbot/strategy/avellaneda_market_making/avellaneda_market_making_config_map.py", line 56, in order_amount_prompt
    min_amount = await minimum_order_amount(exchange, trading_pair)
  File "/home/hummingbot/hummingbot/client/config/config_helpers.py", line 382, in minimum_order_amount
    mid_price = await get_last_price(exchange, trading_pair)
  File "/home/hummingbot/hummingbot/core/utils/market_price.py", line 27, in get_last_price
    last_prices = await module.get_last_traded_prices(**args)
  File "/home/hummingbot/hummingbot/connector/exchange/kucoin/kucoin_api_order_book_data_source.py", line 307, in get_last_traded_prices
    resp_record = [o for o in resp_json["data"]["ticker"] if o["symbolName"] == trading_pair][0]
IndexError: list index out of range
```

- `dev-0.41.0` works fine on both Pure Market Making and Avellaneda MM, but the pair is showing as `BCHSV-USDT` in the bot instead of `BSV-USDT` like on the exchange

![image](https://user-images.githubusercontent.com/73885708/123732365-625b2300-d8cc-11eb-8079-ea4c8809da38.png)

![image](https://user-images.githubusercontent.com/73885708/123738032-a18e7180-d8d6-11eb-90ec-21c0219bff55.png)

![image](https://user-images.githubusercontent.com/73885708/123732385-6ab35e00-d8cc-11eb-9794-4ace1ac14b7a.png)

![image](https://user-images.githubusercontent.com/73885708/123732415-769f2000-d8cc-11eb-9f43-8d822ea3f859.png)

**Release version**
v0.40.0

**Attachments** 
[logs_hummingbot - glenn.log](https://github.com/CoinAlpha/hummingbot/files/6730394/logs_hummingbot.-.glenn.log)

**Discord Handle**
superflow#2909

**Comment by RC-13** [comment]
Similar case: [3521](https://github.com/CoinAlpha/hummingbot/issues/3521)

**Comment by dennisocana** [comment]
Fixed in version 0.42 release
